FlowCrypt/Common UI/AttachmentManager.swift

```diff
@@ -44,8 +44,8 @@ final class AttachmentManager: NSObject {
 }
 
 extension AttachmentManager: AttachmentManagerType {
-    func open(_ attachment: MessageAttachment) {
-        filesManager.saveToFilesApp(file: attachment, from: self)
+    func open(_ file: FileType) {
```

Comment:
Method names should be renamed if they no longer make sense. If it's downloading the file, and we now also have an option to view the file, then this method should be renamed to `saveToDevice` or similar.

Also, should be refactored as an async method so that the error can be thrown up and handled by wherever the action initiated, with a `Task` and a do/try/catch. This is the preferred pattern going forward.